in_test.go

```diff
@@ -40,7 +40,7 @@ func TestGet(t *testing.T) {
 				CommittedDate: time.Time{},
 			},
 			parameters:     resource.GetParameters{},
-			pullRequest:    createTestPR(1, "master", false, false),
+			pullRequest:    createTestPR(1, "master", false, false, []string{"help wanted"}),
```

Comment:
I think that for `in_test.go` we can just pass a `nil` as the labels for the fake pull requests.